bin/opm

```diff
@@ -661,6 +674,67 @@ _EOC_
     #if ($server_build) {
     #err "something bad bad bad.\n";
     #}
+
+    warn "build successful: '$TarBall'\n";
+}
+
+sub do_build_install () {
+=pod
```

Comment:
These comments should be comments instead of POD documentation. Also, they should appear before the sub prototype.